functions/src/sgyfetch.ts

```diff
@@ -182,7 +187,7 @@ export const fetchMaterials = functions.https.onCall(async (data, context) => {
             const pages = responses[4 * i + 2].page;
             const events = responses[4 * i + 3].event;
 
-            sections[course.section_title.split(' ')[0][0]] = {
+            sections[course.section_title.split(' ')[0][0] as SgyPeriod] = {
```

Comment:
Does
```js
sections[course.section_title.split(' ')[0][0]]
```
work for study hall? I'm not sure what `course.section_title` looks like, but taking the first letter of the first word wouldn't work if the title were "Study Hall".